log4j-api/src/main/java/org/apache/logging/log4j/status/StatusLogger.java

```diff
@@ -158,11 +162,26 @@ public class StatusLogger extends AbstractLogger {
 
     /**
      * The name of the system property that can be configured with a {@link java.time.format.DateTimeFormatter} pattern that will be used while formatting the created {@link StatusData}.
+     * <p>
+     * When not provided, {@link Instant#toString()} will be used.
+     * </p>
      *
+     * @see #STATUS_DATE_FORMAT_ZONE
      * @since 2.11.0
      */
     public static final String STATUS_DATE_FORMAT = "log4j2.StatusLogger.DateFormat";
 
+    /**
+     * The name of the system property that can be configured with a time-zone ID (e.g., {@code Europe/Amsterdam}, {@code UTC+01:00}) that will be used while formatting the created {@link StatusData}.
+     * <p>
+     * When not provided, {@link ZoneId#systemDefault()} will be used.
+     * </p>
+     *
+     * @see #STATUS_DATE_FORMAT
+     * @since 2.23.1
+     */
+    static final String STATUS_DATE_FORMAT_ZONE = "log4j2.StatusLogger.DateFormatZone";
```

Comment:
In `3.x` the format of all properties will be `Component.propertyInCamelCase`, so I would already use `StatusLogger.dateFormatZone` here.

Reply:
I've pushed a fix for this. Could you check it again, please?